rules/brand-more-than-logo/rule.md

```diff
@@ -0,0 +1,74 @@
+<script async src="//www.instagram.com/embed.js"></script>
+**Video: Your brand is more than just a logo (40 sec)**
+
+## Key elements of brand identity
+
+### 1. Color Palette: The Emotional Trigger
+Your brand colors evoke specific emotions. Consistent use helps build recognition and trust. Examples include:
+- **Blue:** Trust and stability (e.g., IBM, Visa).
+- **Red:** Energy and excitement (e.g., Coca-Cola, Netflix).
+- **Green:** Growth and tranquility (e.g., Starbucks, Whole Foods).
+
+### 2. Typography: Communicating Personality
+Fonts do more than look good—they communicate your brand’s personality.
+- **Sans-serif fonts** (e.g., Helvetica): Modern and approachable.  
+- **Serif fonts** (e.g., Times New Roman): Traditional and reliable.  
+- **Custom fonts** (e.g., Coca-Cola’s script): Unique to the brand.
+
+### 3. Tone of Voice: Shaping Perception
+How you say things matters just as much as what you say. Examples of tone in action:
+- **Formal:** Financial institutions (e.g., Goldman Sachs).
+- **Conversational:** Consumer apps (e.g., Slack).
+- **Inspirational:** Athletic brands (e.g., Nike).
+
+## Brand guidelines ensure consistency
+
+Without guidelines, brand communication becomes inconsistent. A brand guideline includes:
+- **Color usage** (HEX, RGB values)
+- **Typography standards** (font sizes, spacing)
+- **Logo rules** (placement, acceptable variations)
+- **Imagery guidelines** (photo styles and iconography)
+- **Tone of voice examples** (proper messaging across platforms)
+
+::: info
+#### Case Study: Google's Seamless Brand Identity
+Even without their logo, Google’s identity is recognizable through their playful tone, minimalistic design, and consistent color use. This demonstrates the power of cohesive branding across all touchpoints.
+:::
+
+## A logo is only the beginning
+
+Your logo may be the face of your brand, but it is not the full story. By aligning all brand elements—colors, typography, messaging, and voice—you build a strong, lasting impression that resonates with customers. **Consistency** across every interaction fosters trust and makes your brand instantly recognizable.
+
+**Key Takeaways:**
```

Comment:
@bettybondoc 

Don't think we need this extra sentence, we can show the list straigh away

- [ ] 1. Please remove if agreed

Reply:
@tiagov8 - not sure which sentence you're referring to? :)

Reply:
@tiagov8

Done - removed "Key Takeaways:"